crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -240,6 +242,8 @@ impl CheckpointExecutor {
             info!("Finished syncing and executing checkpoint {}", seq);
         }
 
+        fail_point!("highest-executed-checkpoint");
```

Comment:
up to you but i think it might be more descriptive to name the fail point after the code location where it's failing, e.g. maybe "process-executed-checkpoint"?

"Highest executed checkpoint" is a concept that appears in various places throughout the code so it's not as immediately obvious from that name which thing is being made to fail

Reply:
Oops forgot to address this comment before merging. Will fix separately